docs/modules/ROOT/pages/how-to/choose-node-class.adoc

```diff
@@ -0,0 +1,68 @@
+= Choose Node Class
+
+Some zones of {product} provide two or more node classes you can choose from to schedule your workload.
+For an explanation on what node group are and which one are available, please have a look at xref:references/node-classes.adoc[].
+
+This page will explain to you how you can have your workload scheduled on a specific node class.
+You can do so for all workload within a given namespace.
+But you can also do so for a single pod.
+It's even possible, to a node class as default for a namespace, while selected pods run on another node class.
+
+[CAUTION]
+====
+Your workload might may get stuck with pods in status `Pending` with the following error message:
+
+[source]
+----
+0/X nodes are available: […] X node(s) didn't match Pod's node affinity/selector.
+----
+
+This happens, when you set a node class that doesn't exist on the system.
+====
+
+== Setting Namespace Defaults
+
+You can set the node class on a namespace level by setting the `appuio.io/default-node-selector` annotation on the namespace.
+The value of the annotation is a comma separated list of key value pairs that's expanded to a node selector.
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    appuio.io/default-node-selector: "appuio.io/node-class=plus" <1>
+  name: my-namespace
+----
+<1> The node class to use for workloads in this namespace.
+
+== Setting Workload Node Selector
+
+Single Pods and their controllers can override the node class by setting the `.spec.nodeSelector` field.
```

Comment:
Either write "Pod" in the whole document with uppercase, or in the whole document full lowercase, but please don't mix.